sockeye/output_handler.py

```diff
@@ -140,3 +115,29 @@ def handle(self, t_input: sockeye.inference.TranslatorInput, t_output: sockeye.i
                              t_input.tokens,
                              t_output.tokens,
                              self.threshold)
+
+
+def get_output_handler(output_type: str,
```

Comment:
no, but if you don't mind move it back to the top. The string declaration is exactly to avoid the type error.